velox/vector/FlatVector-inl.h

```diff
@@ -141,6 +141,12 @@ void FlatVector<T>::copyValuesAndNulls(
     const BaseVector* source,
     const SelectivityVector& rows,
     const vector_size_t* toSourceRow) {
```

Comment:
nit: maybe add a short circuit path for 
```
if (!rows.hasSelections()) {
    return;
  }
```
similar to what we have in the other API that takes an index and size

Reply:
This is already done in the caller:

```
void copy(
      const BaseVector* source,
      const SelectivityVector& rows,
      const vector_size_t* toSourceRow) override {
    if (!rows.hasSelections()) {
      return;
    }
    copyValuesAndNulls(source, rows, toSourceRow);
  }
```